<launch>

  <node pkg="exercise1" type="mr1" name="mr1" output="screen">
    <param name="mode" value="-1" />
    <param name="plot_grid" value="true" />
    <param name="plot_motion" value="false" />
    <param name="plot_sensor" value="true" />
    <param name="plot_map" value="false" />
    <param name="plot_robot_on_map" value="false" />
    <param name="nr_of_samples" value="50" />
    <param name="sigmas_motion" value="0.1, 0.1, 0.1, 0.1, 0.1, 0.1" />
    <param name="sigmas_sensor" value="0.1, 0.1, 0.1" />
    <param name="dt" value="2.0" />
    <remap from="laser" to="base_scan" />
    <remap from="cmd" to="cmd_vel" />
    <remap from="odom" to="odom" />
  </node>
</launch>
